extensions/interactions/base_test.py

```diff
@@ -355,80 +355,75 @@ def test_default_interactions_are_valid(self):
         all_interaction_ids = (
             interaction_registry.Registry.get_all_interaction_ids())
         for interaction_id in all_interaction_ids:
+            # TODO(#10212):Ignoring the RatioExpressionInput for now as
+            # there's no learner's flow for the interaction yet.
+            # Once added this needs to be removed.
+
             # Check that the interaction id is valid.
             self.assertTrue(self._is_camel_cased(interaction_id))
             hyphenated_interaction_id = (
                 utils.camelcase_to_hyphenated(interaction_id))
-
             # Check that the interaction directory exists.
             interaction_dir = os.path.join(
                 feconf.INTERACTIONS_DIR, interaction_id)
             self.assertTrue(os.path.isdir(interaction_dir))
-
             # The interaction directory should contain the following files:
             #  Required:
             #    * A python file called {InteractionName}.py.
             #    * An __init__.py file used to import the Python file.
             #    * A TypeScript file called {InteractionName}.ts.
-            #    * A directory name 'directives' containing TS and HTML files
-            #      for directives
-            #    * A directory named 'static' containing at least a .png file.
+            #    * A directory name 'directives' containing TS and HTML
+            #       files for directives
+            #    * A directory named 'static' containing at least a .png
+            #       file.
             #  Optional:
             #    * A JS file called protractor.js.
             interaction_dir_contents = (
                 self._listdir_omit_ignored(interaction_dir))
-
             interaction_dir_optional_dirs_and_files_count = 0
-
             try:
                 self.assertTrue(os.path.isfile(os.path.join(
                     interaction_dir, 'protractor.js')))
                 interaction_dir_optional_dirs_and_files_count += 1
             except Exception:
                 pass
-
             try:
                 self.assertTrue(os.path.isfile(os.path.join(
                     interaction_dir,
-                    '%s-prediction.service.ts' % hyphenated_interaction_id)))
+                    '%s-prediction.service.ts' % hyphenated_interaction_id))) # pylint: disable=line-too-long
```

Comment:
Here and elsewhere, please **do not introduce new pylint pragmas**. We already have enough of them that we're trying to get rid of. If you are unsure about how to avoid a particular pragma, you can consult @Hudda.

Reply:
Also, it is not needed here since the line is shorter than 80 characters.

Reply:
Would be reverting them.